### DISCUSSION TITLE
How to diagnose file provider issue with some files

### DISCUSSION THREAD

**Discussion by stevegroom** [music-assistant/support#3178] [Q&A]
I have a Synology NAS sharing /media/Music to MusicAssistant via SMB. Music Assistant and Music Assistant Beta are alternately running as Add-Ons on Home Assistant yellow. 

The library is organised by artist / album, but a certain amount of bit rot meant that some albums are missing tracks. The Bit Rot is no longer an issue, I am re-ripping CDs to fill in the gaps. At the same time, I am using MusicBrainz Picard to update the tags. 

I ripped about twenty CDs yesterday, removed and recreated the folders on /media/Music and used the refresh option in Music Assistant to complete the update. 

Using MA file browser some of the recreated folders are empty. On the SMB server the files are there. The permissions of the working and missing folders and tracks are identical. 

SMB 3 is be used for the connection. SMB was restarted on the NAS. NAS was also restarted. HA is up to date and restarted. MA and MA beta both have the same fault.

 I have Studio Code and Terminal running, but can't see the MA mounts etc. How do I go about diagnosing further.

**Comment by OzGav** [comment]
I wouldn’t worry so much about the browse function for now. The question is can you find the tracks in the library?
